krampoline/src/types/CommonType/commonType.ts

```diff
@@ -0,0 +1,29 @@
+export interface ChildrenProps {
+  children: React.ReactNode;
+}
+
+export interface RetryButtonProps {
+  children: React.ReactNode;
+  onClick: () => void;
+}
```

Comment:
ChildrenProps가 많이 쓰이고 있는데 아래처럼 상속시켜주면 중복 코드를 줄일 수 있을 것 같아요 !

```suggestion
export interface RetryButtonProps extends ChildrenProps {
  onClick: () => void;
}
```

Reply:
감사합니다 !